### PR TITLE
Update AWS CDK + Default ASG for ECS

### PR DESCRIPTION
**Pull request by timpur**

### CODE REVIEW

File: packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts [hunk 1]
```diff
@@ -75,17 +67,22 @@ export const EcsSolarSystemCoreStackBuilder = (
         ...clusterProps,
         clusterName: this.singletonId('Cluster'),
         vpc: this.vpc,
+        capacity:
+          clusterProps.capacity !== null
```
Comment (jbnaik) [inline packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts:71]: Shouldn't this be  `clusterProps.capacity == null`

Comment (timpur) [reply packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts:71]: nah the idea is if they done want the default asg then pass in null (explicit)

Comment (timpur) [reply packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts:71]: undefined = default and something is merged in with defaults ...

Comment (jbnaik) [reply packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts:71]: ahh...ok so by default ASG will be created

Comment (timpur) [reply packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts:71]: i think if they want their own ASG then pass in null and create your own, atm we have no real requirement to have access to the ASG ... but when that changes, then in the future well change the props ?

**Pull request merged**